### ISSUE TITLE
Android Mp3 player upgrade

### ISSUE THREAD

**Issue by Badcholo** [simeonradivoev/MatterOverdrive#373]
A simple Gui with a reload button to play all your mp3 files from a folder.
Shuffle, repeat, play, pause, play all, stop, next, back,
Or a hand held item. I currently use Icraft, maybe integrade it into an android ability so i don't have to carry it around with me. It would be cool to be able to shop and play music and can be used for much more than just online shopping and such. It would make being an android more awesome to be able to access certain functions from inside your brain. Shop, play music, search for items from your ME system from AE2 mod. Maybe even control an army of androids by collecting parts and crafting your very own android army. 
http://minecraft.curseforge.com/projects/icraft

**Comment by Badcholo** [comment]
I wasn't able to get the teleporter to work, i used dev build latest yesterday, i had an issue where i couldn't type in the coords. If this is a known issue i won't create a new issue. I really like the way you guys have updated it. It's freaking awesome

**Comment by simeonradivoev** [comment]
What do you mean type ? You can only change the coordinates by adding and subtracting with the buttons. The transporter also has a very small range by itself. You need range upgrades to be able to add coordinates a distance away. 

As for your suggestion the mp3 part is quite silly.

**Comment by Badcholo** [comment]
Yes, Silly but very useful. I gotta tell ya, being able to play my music from a mp3 folder without having to exit minecraft has been a great experience for me. specially with icraft, it's like a Iphone in minecraft. It is a actual craftable item that you can use to order pizza, see web pages and play mp3 files. Icraft already has these features and is a very nice mod. There really is no need to integrate it i guess, it would of been cool to access the icraft gui from a push of a button instead of having to get the item out of your back pack, but i guess that way it is more real.

**Comment by Badcholo** [comment]
The latest dev version does not have the + and - buttons. I was able to type in the coords by selecting the slot. It didn't work every time thou. I'll upload some screen shots when i get home in 20min or so.
